plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/type/UnsupportedTypeException.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery.type;
+
+import com.google.cloud.bigquery.StandardSQLTypeName;
+
+/**
+ * Unsupported type exception occurs only for data types
+ * known to be not supported, like STRUCT, not all unknown tokens.
+ */
+public class UnsupportedTypeException
```

Comment:
This name looks little confusing because STRUCT type is supported except for parsing. How about renaming to ParsingException or something?

Reply:
It's not supported in the `type` package. `ParsingException` seems to be too broad, or I'd have to refactor the exception to use it instead of `IllegalArgumentException` and make the `typeName` parameter optional. I don't think it's worth doing.

Reply:
> It's not supported in the type package

I know. However, the class is used from outside of the package (BigQueryTypeManager) either.

Reply:
How can I make it more clear, using a fully qualified name, or with a comment?